pkgs/by-name/ca/cabin/package.nix

```
@@ -1,46 +1,73 @@
 {
   lib,
-  stdenvNoCC,
+  stdenv,
   fetchFromGitHub,
+  tbb_2021_11,
+  libgit2,
+  curl,
+  fmt,
+  nlohmann_json,
+  pkg-config,
 }:
 
-stdenvNoCC.mkDerivation rec {
+let
+  toml11 = fetchFromGitHub rec {
+    owner = "ToruNiina";
+    repo = "toml11";
+    version = "4.2.0";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-NUuEgTpq86rDcsQnpG0IsSmgLT0cXhd1y32gT57QPAw=";
+  };
+in
+stdenv.mkDerivation rec {
   pname = "cabin";
-  version = "1.005";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
-    owner = "impallari";
-    repo = "Cabin";
-    rev = "982839c790e9dc57c343972aa34c51ed3b3677fd";
-    hash = "sha256-9l4QcwCot340Bq41ER68HSZGQ9h0opyzgG3DG/fVZ5s=";
+    owner = "cabinpkg";
+    repo = pname;
+    rev = "refs/tags/${version}";
```

```suggestion
    tag = version;
```

I think this is a new thing we can do.